### ISSUE TITLE
DateTime issue

### ISSUE THREAD

**Issue by metalicsmell** [PSAppDeployToolkit/PSAppDeployToolkit#63]
Using latest script.
```
At C:\Users\User\Desktop\Java\AppDeployToolkit\AppDeployToolkitMain.ps1:63 char:1
+ [datetime]$currentDate = Get-Date -UFormat '%d-%m-%Y'
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : MetadataError: (:) [], ArgumentTransformationMetadataException
    + FullyQualifiedErrorId : RuntimeException 
```

Only after replacing to:

```$currentDate = Get-Date -UFormat '%d-%m-%Y'```

it starts working correctly.

One more thing: Java x64 not removing while ```RemoveMSI-Application 'Java'```, in same time x86 removed correctly. Also, no logFile in %logdir% - lokks like Java x64 not visible for script.

**Comment by mmashwani** [comment]
I have fixed the $currentDate issue in the latest beta. I am closing this issue. Please create a separate issue for the RemovMSI-Application issue with more details.
